### PR TITLE
[9.0][ADD] purchase_request_department

### PR DESCRIPTION
**Pull request by gmeficent**
Purchase Request Department
===========================

This module adds the user department in a new field in the purchase request
form and allows to group by department on the tree view.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review :+1:

**Comment by JordiBForgeFlow** [comment]
@gmeficent Can you add an init_hook that populates the departments for all existing purchase requests when the module is installed?

**Comment by JordiBForgeFlow** [comment]
Can you add the department to the purchase request lines, as a related field to the purchase request? To be shown in the purchase request tree view.

**Comment by JordiBForgeFlow** [comment]
@gmeficent you have some travis errors. Can you review?

Please add also some tests.

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: technical and functional review

**Review by MiquelRForgeFlow — APPROVED** [review]
nice :+1:

File: purchase_request_department/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
```
Comment (AaronHForgeFlow) [inline purchase_request_department/__init__.py]: Should it be 2017?

**Review by AaronHForgeFlow — COMMENTED** [review]
It works good overall but I get an error on runbot in the Purchase Request Lines when grouping by department:

image

File: purchase_request_department/__init__.py (hunk 1 above)
Comment (serpentcs-dev1) [reply purchase_request_department/__init__.py]: @gmeficent As far as I know it should be 2017

**Comment by serpentcs-dev1** [comment]
Code and functional review:+1: 

Works fine except the group by in request line as @aheficent mentioned.

**Comment by gmeficent** [comment]
Thanks, I'll check again :+1: 
[EDIT] it should be alright now :)

**Review by serpentcs-dev1 — APPROVED** [review]
LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**